diff -ur glib-2.34.3/configure glib-2.34.3.patched/configure
--- glib-2.34.3/configure	2012-11-27 05:08:50.000000000 +1000
+++ glib-2.34.3.patched/configure	2013-02-11 20:24:33.785082316 +1000
@@ -7245,19 +7245,6 @@
 fi
 
 fi
-if test -n "$PKG_CONFIG"; then
-	_pkg_min_version=0.16
-	{ $as_echo "$as_me:${as_lineno-$LINENO}: checking pkg-config is at least version $_pkg_min_version" >&5
-$as_echo_n "checking pkg-config is at least version $_pkg_min_version... " >&6; }
-	if $PKG_CONFIG --atleast-pkgconfig-version $_pkg_min_version; then
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: result: yes" >&5
-$as_echo "yes" >&6; }
-	else
-		{ $as_echo "$as_me:${as_lineno-$LINENO}: result: no" >&5
-$as_echo "no" >&6; }
-		PKG_CONFIG=""
-	fi
-fi
 
 if test "x$enable_debug" = "xyes"; then
   if test x$cflags_set != xset ; then
