diff -ur glib-2.34.3/glib/tests/Makefile.in glib-2.34.3.patched/glib/tests/Makefile.in
--- glib-2.34.3/glib/tests/Makefile.in	2012-11-27 05:08:54.000000000 +1000
+++ glib-2.34.3.patched/glib/tests/Makefile.in	2013-02-11 20:38:28.637200940 +1000
@@ -56,7 +56,7 @@
 	$(top_srcdir)/Makefile.decl $(top_srcdir)/depcomp \
 	$(top_srcdir)/mkinstalldirs
 noinst_PROGRAMS = $(am__EXEEXT_3) test-spawn-echo$(EXEEXT)
-@HAVE_GCC_TRUE@am__append_1 = -Wstrict-aliasing=2
+#@HAVE_GCC_TRUE@am__append_1 = -Wstrict-aliasing=2
 @HAVE_EVENTFD_TRUE@am__append_2 = gwakeup-fallback
 @OS_UNIX_TRUE@am__append_3 = unix-multithreaded unix unix-nothreads \
 @OS_UNIX_TRUE@	include
