Index: b/build/autoconf/config.guess
===================================================================
--- a/build/autoconf/config.guess
+++ b/build/autoconf/config.guess
@@ -1023,6 +1023,9 @@ EOF
     ppc64:Linux:*:*)
 	echo powerpc64-unknown-linux-${LIBC}
 	exit ;;
+    ppc64le:Linux:*:*)
+	echo powerpc64le-unknown-linux-gnu
+	exit ;;
     ppc:Linux:*:*)
 	echo powerpc-unknown-linux-${LIBC}
 	exit ;;
