From 4ea74a440d6f7ff0f6e2d25a780aee21f86762a8 Mon Sep 17 00:00:00 2001
From: Daniel Veillard <veillard@redhat.com>
Date: Mon, 29 Oct 2012 10:27:18 +0800
Subject: [PATCH] Fix a portability issue for GCC < 3.4.0

---
 timsort.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/timsort.h b/timsort.h
index b3a32f0..99697a0 100644
--- a/timsort.h
+++ b/timsort.h
@@ -48,7 +48,7 @@ typedef unsigned __int64 uint64_t;
 int compute_minrun(uint64_t);
 
 #ifndef CLZ
-#ifdef __GNUC__
+#if defined(__GNUC__) && ((__GNUC__ == 3 && __GNUC_MINOR__ >= 4) || (__GNUC__ > 3))
 #define CLZ __builtin_clzll
 #else
 
-- 
1.7.11.7
