--- a/lib/config.hin	2025-01-24 14:12:21.539591363 -0500
+++ b/lib/config.hin	2025-01-24 14:12:45.081684749 -0500
@@ -197,7 +197,7 @@
 
 /* Enable compile-time and run-time bounds-checking, and some warnings,
       without upsetting newer glibc. */
-   #if defined __OPTIMIZE__ && __OPTIMIZE__
+   #if !defined _FORTIFY_SOURCE && defined __OPTIMIZE__ && __OPTIMIZE__
    # define _FORTIFY_SOURCE 2
    #endif
   
