This breaks on really old pkg-config versions.

diff -ur xz-5.0.4/src/liblzma/liblzma.pc.in xz-5.0.4.patched/src/liblzma/liblzma.pc.in
--- xz-5.0.4/src/liblzma/liblzma.pc.in  2012-06-22 16:22:30.000000000 +1000
+++ xz-5.0.4.patched/src/liblzma/liblzma.pc.in  2013-02-13 22:13:57.362082954 +1000
@@ -12,7 +12,6 @@
 
 Name: liblzma
 Description: General purpose data compression library
-URL: @PACKAGE_URL@
 Version: @PACKAGE_VERSION@
 Cflags: -I${includedir}
 Libs: -L${libdir} -llzma
